config/settings.yml

```diff
@@ -0,0 +1,2 @@
+marmotta:
+  url: 'http://localhost:8983/marmotta'
```

Comment:
Should probably get this to work. Does Sufia already have a YML configuration file? Maybe install RailsConfig?